src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -509,7 +509,8 @@ private void InitParallelParameterSet()
         private void ProcessParallelParameterSet()
         {
             // Validate piped InputObject
-            if (_inputObject.BaseObject is ScriptBlock)
+            if (_inputObject != null && 
+                _inputObject.BaseObject is ScriptBlock)
```

Comment:
The error message for null does not look correct.

Reply:
A null input object is valid, so there is no error for that.  This check just prevents a null dereference when checking the input type.